sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Program.cs

```diff
@@ -0,0 +1,132 @@
+            }
+
+            if (runEventBufferedProducerTest)
+            {
+                var testConfiguration = new ProducerConfiguration();
+                environment.TryGetValue("EVENTHUB_NAMESPACE_CONNECTION_STRING", out testConfiguration.EventHubsConnectionString);
+                environment.TryGetValue("APPINSIGHTS_INSTRUMENTATIONKEY", out testConfiguration.InstrumentationKey);
+                environment.TryGetValue("EVENTHUB_NAME_EBPT", out testConfiguration.EventHub);
+                environment.TryGetValue("EVENTHUB_PARTITIONS_EBPT", out testConfiguration.NumPartitions);
+                PromptForResources(testConfiguration, "EventBufferedProducerTest");
+
+                testConfiguration.ConcurrentSends = 1;
+
+                var testRun = new BufferedProducerTest(testConfiguration);
+                await testRun.Run();
+            }
+
+            if (runConcurrentBufferedProducerTest)
+            {
+                var testConfiguration = new ProducerConfiguration();
+                environment.TryGetValue("EVENTHUB_NAMESPACE_CONNECTION_STRING", out testConfiguration.EventHubsConnectionString);
+                environment.TryGetValue("APPINSIGHTS_INSTRUMENTATIONKEY", out testConfiguration.InstrumentationKey);
+                environment.TryGetValue("EVENTHUB_NAME_CBPT", out testConfiguration.EventHub);
+                environment.TryGetValue("EVENTHUB_PARTITIONS_CBPT", out testConfiguration.NumPartitions);
+                PromptForResources(testConfiguration, "ConcurrentBufferedProducerTest");
+
+                testConfiguration.ConcurrentSends = 5;
+
+                var testRun = new BufferedProducerTest(testConfiguration);
+                await testRun.Run();
+            }
+
+            if (runBurstBufferedProducerTest)
+            {
+                var testConfiguration = new ProducerConfiguration();
+                environment.TryGetValue("EVENTHUB_NAMESPACE_CONNECTION_STRING", out testConfiguration.EventHubsConnectionString);
+                environment.TryGetValue("APPINSIGHTS_INSTRUMENTATIONKEY", out testConfiguration.InstrumentationKey);
+                environment.TryGetValue("EVENTHUB_NAME_BBPT", out testConfiguration.EventHub);
+                environment.TryGetValue("EVENTHUB_PARTITIONS_BBPT", out testConfiguration.NumPartitions);
+                PromptForResources(testConfiguration, "BurstBufferedProducerTest");
+
+                testConfiguration.ConcurrentSends = 5;
+                testConfiguration.ProducerPublishingDelay = TimeSpan.FromMinutes(15);
+
+                var testRun = new BufferedProducerTest(testConfiguration);
+                await testRun.Run();
+            }
+        }
+
+        private static void PromptForResources(ProducerConfiguration testConfiguration, string testName)
+        {
+            // Prompt for the App Insights instrumentation key
+            while (string.IsNullOrEmpty(testConfiguration.InstrumentationKey))
```

Comment:
Not sure this works well in the stress environment;  should we consider locking this down to some kind of -`--interactive` flag and throwing if we don't have enough information otherwise?